assets/js/page-enhancements.js

```diff
@@ -18,9 +18,19 @@
     $('#find-recipient-by-prison-number-form #prisonNumber').focus()
   })
 })($, document)
-;(autoFocusPrisonNameField = ($, document) => {
+;(autoFocusPrisonerNameField = ($, document) => {
   $(document).ready(() => {
-    $('#create-new-contact-form #prisonerName').focus()
+    $('#find-recipient-by-prisoner-name-form #prisonerName').focus()
+  })
+})($, document)
+;(autoFocusContactPrisonNameField = ($, document) => {
+  $(document).ready(() => {
+    $('#create-new-contact-by-prison-number-form #prisonerName').focus()
+  })
+})($, document)
+;(autoFocusPrisonerDobField = ($, document) => {
+  $(document).ready(() => {
+    $('#create-new-contact-by-prisoner-name-form #prisonerDob-day').focus()
   })
```

Comment:
Out of interest, does this autofocus approach work with IE? If so, I might steal it.

Reply:
Apparently it does: https://caniuse.com/autofocus

Reply:
Ooh sorry that was the HTML focus attribute - which we had mixed results with. This gives a good breakdown of the various ways of focusing and compatibility: https://caniuse.com/?search=focus